OpenRA.Mods.Common/Traits/World/SubterraneanActorLayer.cs

```diff
@@ -15,6 +15,7 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	[TraitLocation(SystemActors.World | SystemActors.EditorWorld)]
```

Comment:
```suggestion
	[TraitLocation(SystemActors.World)]
```

imho as that is a runtime feature.

Reply:
Good point, done.